src/dmd/parse.d

```diff
@@ -5523,6 +5523,11 @@ final class Parser(AST) : Lexer
                     s = null; // don't propagate parsing errors
                 break;
             }
+
+        case TOK.else_:
+            error("found `else` without a corresponding `if`, `version` or `debug` statement");
```

Comment:
Do we want to include `static if` in the error message?

Reply:
`static if` is covered by `if`